tests/unittests/tests-base64/tests-base64.c

```diff
@@ -303,13 +303,13 @@ static void test_base64_07_stream_decode(void)
 
 static void test_base64_08_encode_16_bytes(void)
 {
-    const int buffer_size = 16;
+    enum { buffer_size = 16 };
```

Comment:
was suggested by @Kijewski and after some lookup I also found in several blogs online that this is (besides using magic numbers or `#define`) the way to go.